torch/_inductor/codegen/common.py

```diff
@@ -592,21 +592,28 @@ def func(x, y):
 
         for funcname, data in pointwise_overrides_data.items():
             impl = getattr(data, target)
+            if impl is None:
+                continue
+
             if isinstance(impl, str):
                 nof_args = 2 if "{y}" in impl else 1
                 # extend the following dictionary with factory
                 # functions for a specific number of arguments as
                 # needed:
                 factory = {1: pointwise_factory_1, 2: pointwise_factory_2}[nof_args]
-                setattr(cls, funcname, staticmethod(factory(impl)))
+                impl = factory(impl)
+
+            setattr(cls, funcname, staticmethod(impl))
 
 
 @dataclasses.dataclass
 class OverridesData:
     name: str
-    cpp: str
-    triton: Optional[str] = None  # None when not impl in libdevice/triton
-    cppvec: Optional[str] = None  # None when not impl in aten/.../vec
+    cpp: Union[str, Callable[..., str]]
+    # None when not impl in libdevice/triton
+    triton: Union[Optional[str], Callable[..., str]] = None
+    # None when not impl in aten/.../vec
+    cppvec: Union[Optional[str], Callable[..., str]] = None
```

Comment:
May I know why you chose to maintain codegen backend specific things in the common place instead of adding them to their respective subclasses as the original design does?

Reply:
It has simplified adding a new `ops` method into a 7 line change that only touches a single file, which I think is pretty compelling. It does mean there are more places you need to look to find the overload which may be annoying, but it's still very grep-able so I don't think this is a major issue.

Also note that this just adds a method onto the `{Backend}Overrides` class in an unconventional way, rather than replacing the overrides mechanism.